scripts/bundleSize/index.js

```diff
@@ -1,258 +1,264 @@
 #! /usr/bin/env node
+/* eslint-disable import/extensions */
 /* eslint-disable no-console */
 
-const ora = require('ora');
-const chalk = require('chalk');
-const Table = require('cli-table');
-const sortByBundlesTotalAscending = require('./sortByBundlesTotalAscending');
-const getAverageBundleSize = require('./getAverageBundleSize');
-const createConsoleError = require('./createConsoleError');
-const {
+import ora from 'ora';
+import chalk from 'chalk';
+import Table from 'cli-table';
+import sortByBundlesTotalAscending from './sortByBundlesTotalAscending.js';
+import getAverageBundleSize from './getAverageBundleSize.js';
+import createConsoleError from './createConsoleError.js';
+import {
   getPageBundleData,
   getServiceConfigBundleData,
   getServiceThemeBundleData,
-} = require('./getBundleData');
-const { MIN_SIZE, MAX_SIZE } = require('./bundleSizeConfig');
-
-const bundleType = process.env.bundleType || 'modern';
-const serviceConfigBundleData = sortByBundlesTotalAscending(
-  getServiceConfigBundleData(),
-);
-const serviceThemeBundleData = sortByBundlesTotalAscending(
-  getServiceThemeBundleData(),
-);
-const serviceConfigBundlesTotals = serviceConfigBundleData.map(
-  ({ totalSize }) => totalSize,
-);
-const serviceThemeBundlesTotals = serviceThemeBundleData.map(
-  ({ totalSize }) => totalSize,
-);
-const smallestServiceConfigBundleSize = Math.min(...serviceConfigBundlesTotals);
-const largestServiceConfigBundleSize = Math.max(...serviceConfigBundlesTotals);
-const averageServiceConfigBundleSize = getAverageBundleSize(
-  serviceConfigBundlesTotals,
-);
-
-const smallestServiceThemeBundleSize = Math.min(...serviceThemeBundlesTotals);
-const largestServiceThemeBundleSize = Math.max(...serviceThemeBundlesTotals);
-const averageServiceThemeBundleSize = getAverageBundleSize(
-  serviceThemeBundlesTotals,
-);
+} from './getBundleData.js';
+import { MIN_SIZE, MAX_SIZE } from './bundleSizeConfig.js';
+export default () => {
```

Comment:
Changed the `bundleSize/index.js` to no longer run the code inline but expose it as a default export. This allows it to be consumed in the test as an ESM module.